packs/feats/reptile-rider.json

```
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>To you, crocodilians, giant lizards, and dinosaurs are loyal steeds, not monsters. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Ride] feat, even if you don't meet the prerequisites. You gain a +1 circumstance bonus to Nature checks to Handle an Animal as long as the animal is a reptile, dinosaur, or even a non-sapient dragon.</p>\n<hr />\n<p><em><strong>Note</strong> This feat refers to Nature checks to Handle an Animal. This check does not exist in Pathfinder 2, and so the feat instead gives a bonus to Nature checks to @UUID[Compendium.pf2e.actionspf2e.Item.Command an Animal].</em> </p>"
+            "value": "<p>To you, crocodilians, giant lizards, and dinosaurs are loyal steeds, not monsters. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Ride] feat, even if you don't meet the prerequisites. You gain a +1 circumstance bonus to Nature checks to Handle an Animal as long as the animal is a reptile, dinosaur, or even a non-sapient dragon.</p>\n<hr />\n<p><em><strong>Note</strong> This feat refers to Nature checks to Handle an Animal. This check does not exist in Pathfinder 2, and so the feat instead gives a bonus to Nature checks to @UUID[Compendium.pf2e.actionspf2e.Item.Command an Animal].</em></p>"
```

```suggestion
            "value": "<p>To you, crocodilians, giant lizards, and dinosaurs are loyal steeds, not monsters. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Ride] feat, even if you don't meet the prerequisites. You gain a +1 circumstance bonus to Nature checks to @UUID[Compendium.pf2e.actionspf2e.Item.Command an Animal] as long as the animal is a reptile, dinosaur, or even a non-sapient dragon.</p>"
```
I'm fine fixing the obvious typo.